.travis.yml

```diff
@@ -1,4 +1,7 @@
 sudo: required
+os:
+  - linux
```

Comment:
I suggest to move `dist: trusty` under `- linux`.

Reply:
Unfortunately, that's not a valid syntax :disappointed: See https://docs.travis-ci.com/user/multi-os/

We could eventually do it in the `matrix/include` context, but I just tried and it does not play well with multiple gemfiles and ruby versions like we do (we would have to explicitly declare all permutations in the `matrix/include` context, apparently).

Reply:
OIC, so keep it like that :+1: